cpp/cmake/thirdparty/get_rmm.cmake

```diff
@@ -16,9 +16,11 @@
 
 function(find_and_configure_rmm VERSION)
 
-    if(TARGET rmm::rmm)
-        return()
-    endif()
+    # No way to override the version rapids-cmake pulls
+    # so we need to manually checkout when needing a diverged
+    # version
+    # include(${rapids-cmake-dir}/cpm/rmm.cmake)
+    # rapids_cpm_rmm()
```

Comment:
We can remove the pinned rmm now (I'm doing it in #322 as well to run CI correctly )

Reply:
done